bash-preexec.sh

```diff
@@ -293,12 +260,13 @@ __bp_install_after_session_init() {
         return 1;
     fi
 
-    local existing_prompt_command
-    existing_prompt_command=$(__bp_prompt_command_with_semi_colon)
+    if [[ -n "$PROMPT_COMMAND" ]]; then
+        eval "original_prompt_command() { $PROMPT_COMMAND; }"
```

Comment:
Small edge case here. I have a semi colon previously at the end of my prompt command which errors in this version with:

```bash
bash: syntax error near unexpected token `;;'
```
Any ideas to address?